core/main/client/browser.js

```diff
@@ -3919,8 +3919,8 @@ beef.browser = {
         var results;
 
         function unique(array) {
-          return $.grep(array, function(el, index) {
-            return index === $.inArray(el, array);
+          return $j.grep(array, function(el, index) {
+            return index === $j.inArray(el, array);
```

Comment:
Looks good - just testing it on my end now. Any idea why this works in Firefox and not Chrome? It's supposed to work (as far as compatibility is concerned) as far back as pre IE-7. 

I'm assuming this specific file needed to call the $j because that's been assigned jQuery.noConflict() in `core/main/client`. It's not needed in the other client-side code that was changed?

Reply:
Confirmed to be working in Chrome + Firefox on Kali:

![image](https://user-images.githubusercontent.com/46417690/87024612-58b24a00-c1a7-11ea-858d-586b1612427d.png)

Reply:
> Any idea why this works in Firefox and not Chrome?

None. Don't know. Don't care.

> It's not needed in the other client-side code that was changed?

It probably is needed, yes.

If we need to define functions to unique an array, they should probably be made available as a core hook API function for use elsewhere.

> Thanks for fixing my stuff-up - I'll make sure to test compatibility better next time.

I approved it, but absolve myself of responsibility on the grounds that I'm only here to keep the project alive and occasionally warn contributors when they're doing something misguided.

The code is buggy as heck. This is largely due to evolving since before jQuery even existed in addition to trying to support every web browser since 2005. I still believe this is a worthwhile goal and can be improved by implementing a staged hook. But that is a huge piece of work and requires research into the best approach.

Reply:
> The code is buggy as heck. This is largely due to evolving since before jQuery even existed in addition to trying to support every web browser since 2005. I still believe this is a worthwhile goal and can be improved by implementing a staged hook. But that is a huge piece of work and requires research into the best approach.

Yeah, it doesn't have a lack of issues to resolve. I'd be interested to chat more about the staged hook at some point.